airflow/providers/elasticsearch/provider.yaml

```diff
@@ -158,10 +158,19 @@ config:
       index_patterns:
         description: |
           Comma separated list of index patterns to use when searching for logs (default: `_all`).
+          The index_patterns_callable takes precedence over this.
         version_added: 2.6.0
         type: string
         example: something-*
         default: "_all"
+      index_patterns_callable:
+        description: |
+          A string representing the full path to the Python callable path which accept TI object and
+          return comma separated list of index patterns. This will takes precedence over index_patterns.
+        version_added: 2.9.0
```

Comment:
Not core version. ES provider version. Probably 5.4?

Reply:
Those are just hold overs from when this config was in core. My immediate reaction is it should be changed to the version it was added into the provider, but I haven't given this problem much thought.